src/uproot/_util.py

```diff
@@ -520,7 +515,7 @@ def awkward_form(
                     '"float64"'
                 )
             else:
-                raise AssertionError("{0}: {1}".format(repr(model), type(model)))
+                raise AssertionError(f"{repr(model)}: {type(model)}")
```

Comment:
```suggestion
                raise AssertionError(f"{model!r}: {type(model)}")
```

Reply:
These `!r` formatters were available in Python 2, also, but I hadn't been using them. I guess I should.

Reply:
They are more important for readability when you have to nest the `repr()` inside the brackets, but yes, they are likely still slightly nicer for `.format` - you can use them to keep the formatting part (showing a repr) separate from the data part.

Reply:
I used `git ls-files | sed -i 's/{repr(\([^)]*\))}/{\1!r}/g' to clean all of these up.